plugins/module_utils/utils.py

```diff
@@ -366,8 +368,8 @@ def get_all_data_with_pagination(ome_obj, uri, query_param=None):
             resp = ome_obj.invoke_request('GET', next_uri_query)
             report_list.extend(resp.json_data.get("value"))
             next_uri = resp.json_data.get("@odata.nextLink", None)
-    except Exception:
-        resp, report_list = None, []
+    except (URLError, HTTPError, SSLValidationError, ConnectionError, TypeError, ValueError) as err:
```

Comment:
This way it will raise error and return the same to the main module file.
Where we will pass the failed/unreachable parameters. 
Otherwise we wont be able to pass those.